CONTRIBUTING.md

```diff
@@ -51,9 +51,8 @@ https://konghq.com/kong-enterprise-edition/ or contact us at
 There are several channels where you can get answers from the community
 or the maintainers of this project:
 
-- The mailing list, hosted on the [konglayer Google
-  Group](https://groups.google.com/forum/#!forum/konglayer) for asynchronous
-  and lengthy chatter
+- Our public forum, [Kong Nation](https://discuss.konghq.com/), for asynchronous
+  and lengthy chatter.
```

Comment:
pedantic: you will notice that none of the other bullets in this list have a trailing dot